Specalc.package/ButtonModel.extension/instance/beButton.for..st

```diff
@@ -0,0 +1,7 @@
+*Specalc
+beButton: anInteger for: calculator
```

Comment:
For the digit buttons, repeating initialization 3 times (no less 9!) would be enough for me to move the setup into a domain object. Some common choices are:
- create a custom button subclass
- use an extension method (as above) to inject your code into a kernel object without dirtying the kernel. In this case, you want to make sure the method names are unique so they don't clash with other people's extensions, so #beButton:for: is not as good as e.g. #beButton:forSpecalc: